### ISSUE TITLE
Error when i open the program.

### ISSUE THREAD

**Issue by AnarkoDX** [KeyPuncher/WiinUPro#118]
hi , i have this problem when i try to open the program, help would be much appreciated, thanks

![wiinusoft](https://user-images.githubusercontent.com/75274357/222939896-4434bcaf-59cd-41f9-8ed3-c9758d178331.jpg)

**Comment by KeyPuncher** [comment]
It looks like there's an XML parsing issue, likely with the prefs.config file. Take a look at that, should be in the same location as WiinUSoft.exe, if not it should then be in the %APPDATA% folder.

**Comment by AnarkoDX** [comment]
hi , thanks for the help,  in the appdata folder theres nothing from wiinusoft , the file that is with the executable  is this .
![prefs](https://user-images.githubusercontent.com/75274357/222975623-135e0925-aa91-4409-a3de-64920029c74d.jpg)

**Comment by KeyPuncher** [comment]
Very odd. No issues here. First time starting WiinUSoft?
Only other place I can think of that would throw this exception is the parsing of a controller profile.

**Comment by AnarkoDX** [comment]
it used to work before , but i had a problem with gamepass and i had to do an update of windows through an windows iso and that fixed the gamepass bug but i think it ruined this...

**Comment by KeyPuncher** [comment]
Bummer. Have you already tried uninstalling and removing all WiinUSoft files and starting a new?

**Comment by AnarkoDX** [comment]
yes  ,i tried uninstalling and a portable , but still doesnt work ,but i found on event viewer that it has some problem with kernellbase.dll .
